### PR TITLE
feat: logger

### PR DESCRIPTION
**Pull request by 0xkenj1**
# 🤖 Linear

Closes GIT-116

## Description
Replace `console` calls with shared logger.

## Checklist before requesting a review

-   [x] I have conducted a self-review of my code.
-   [x] I have conducted a QA.
-   [x] If it is a core feature, I have included comprehensive tests.

### CODE REVIEW

File: apps/processing/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules
+.git
+.gitignore
+*.md
+generated
```
Comment (jahabeebs) [inline apps/processing/.dockerignore]: will we have a .env in this dir?

File: packages/pricing/src/providers/coingecko.provider.ts [hunk 2]
```diff
@@ -111,19 +113,21 @@ export class CoingeckoProvider implements IPricingProvider {
...
                     throw new NetworkException(error.message, error.status!);
                 }
             }
-            console.error(error);
-            throw new UnknownPricingException(
-                isNativeError(error) ? error.message : JSON.stringify(error),
-                isNativeError(error) ? error.stack : undefined,
-            );
+            const errorMessage =
+                `Coingecko API error: failed to fetch token price` +
```
Comment (jahabeebs) [inline packages/pricing/src/providers/coingecko.provider.ts]: maybe leave a blank space after price so it's separated from the stringified error

**Review by jahabeebs — DISMISSED** [review]
👍🏻

**Review by 0xnigir1 — COMMENTED** [review]
good to go for me after addressing beebs comments, thanks for the logger btw 😄

File: apps/processing/.dockerignore (hunk 1 above)
Comment (0xkenj1) [reply apps/processing/.dockerignore]: we won't

**Review by jahabeebs — APPROVED** [review]

**Review by 0xnigir1 — APPROVED** [review]

**Pull request merged**